### ISSUE TITLE
[FEATURE SUGGESTION] Make default track image the artist's profile picture instead of Soundnode icon

### ISSUE THREAD

**Issue by wmayner** [Soundnode/soundnode-app#550]
Title says it all: when a track has no associated image, the image should default to the artist's profile picture rather than the Soundnode icon.

I'll try and submit a PR for this when I get a moment. Thanks for the app!

**Comment by velagand** [comment]
Yeah I was going to suggest this and found that you already had. I like how the soundcloud website defaults to the artist's profile picture, I think this app should do that as well, it looks nicer than having the SoundNode logo everywhere, especially for the artist profiles that don't upload pictures for any of their tracks.
